Diagnostics/Get-NetView.PS1

```diff
@@ -1024,15 +1030,17 @@ function MellanoxGetFiles {
         $path = [System.Environment]::ExpandEnvironmentVariables("%HOMEDRIVE%\Windows\Temp\MLX5_Dump_Me_Now")
     }
 
-    if((Get-NetAdapter -name $NicName).InterfaceDescription -like "*Mellanox*Virtual*Adapter*") {
+    if ((Get-NetAdapter -name $NicName).InterfaceDescription -like "*Mellanox*Virtual*Adapter*") {
         $locationInformationString = (Get-NetAdapterHardwareInfo -name $NicName).LocationInformationString
-        $slot = $locationInformationString.Split(' ')[$locationInformationString.Split(' ').IndexOf('Slot') + 1]
+
+        $slot   = $locationInformationString.Split(' ')[$locationInformationString.Split(' ').IndexOf('Slot') + 1]
         $serial = $locationInformationString.Split(' ')[$locationInformationString.Split(' ').IndexOf('Serial') + 1]
-        $path += "-" + $slot + "-" + $serial + "-0"
+
+        $path  += "-" + $slot + "-" + $serial + "-0"
```

Comment:
Avoid concatenating strings with + operator to avoid type issues. E.g. `1 + "2" == 3` but `"2" + 1 == "21"`.